README.md

```diff
@@ -32,9 +34,87 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
+```
+
+If you're deploying StackRox on a small node like a local development cluster, run the following command to reduce StackRox resource requirements. Please don't run this command against a production cluster.
+```sh
+helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-services \
+  --set central.resources.requests.memory=1Gi \
+  --set central.resources.requests.cpu=1 \
+  --set central.resources.limits.memory=4Gi \
+  --set central.resources.limits.cpu=1 \
+  --set scanner.autoscaling.disable=true \
+  --set scanner.replicas=1 \
+  --set scanner.resources.requests.memory=500Mi \
+  --set scanner.resources.requests.cpu=500m \
+  --set scanner.resources.limits.memory=2500Mi \
+  --set scanner.resources.limits.cpu=2000m
+```
+To create a secured cluster, you first need to generate an init bundle containing initialization secrets. The init bundle will be saved in `stackrox-init-bundle.yaml`. Use it to provision secured clusters as shown below.
+```sh
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "$(cat stackrox-admin-password.txt)" \
+  central init-bundles generate stackrox-init-bundle --output - > stackrox-init-bundle.yaml
+```
+Then install stackrox-secured-cluster-services in the same cluster using this command with the init bundle you just generated:
+```sh
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services \
+  -f stackrox-init-bundle.yaml \ 
+  --set clusterName=<name_of_the_secured_cluster> 
+```
+Make sure to provide some name in `clusterName` argument meaningful to you. The cluster will be identified by this name in clusters list in StackRox UI.
+
+When deploying stackrox-secured-cluster-services on a different cluster, you will also need to specify the endpoint (address and port number) of Central via `--set centralEndpoint=<endpoint_of_central_service>` command-line argument.
+
+When deploying StackRox on a small node, add the following arguments to the helm install command above. This should reduce stackrox-secured-cluster-services resource requirements. Don't include these when deploying on a production cluster.
+```sh
+--set sensor.resources.requests.memory=500Mi \
+--set sensor.resources.requests.cpu=500m \
+--set sensor.resources.limits.memory=500Mi \
+--set sensor.resources.limits.cpu=500m
+```
+
+To further customize your Helm installation consult these documents:
+* <https://docs.openshift.com/acs/installing/installing_helm/install-helm-quick.html>
+* <https://docs.openshift.com/acs/installing/installing_helm/install-helm-customization.html>
+
+### First use
+
+Follow these steps to get to StackRox UI
+
+1.Setup port forward to central:
+```
+kubectl -n stackrox port-forward deploy/central 8000:8443
+```
+
+2.Open <https://localhost:8000> in your browser.
+
+3.Accept certificate warnings and proceed.
```

Comment:
How about we leave out this step?
Depending on the setup, the user might not even encounter a cert warning (e.g. when adding their own CA in the setup process).
Also, we're kind of motivating users to do something potentially dangerous here (click away this warning).
I would hope that users are aware of the implications, so what do you think?

Reply:
Okay I can leave it out but in that case I think we should also do what we can to avoid creating those warnings in the first place.

Reply:
Suggestion:
If a certificate warning is displayed, you can accept the warnings to install the default self-signed certificate. To configure custom certificates, see the section on "Adding Custom Certificates" in [docs.openshift.com/acs/configuration/add-custom-certificates.html](https://docs.openshift.com/acs/configuration/add-custom-certificates.html).

Reply:
Went with Michaels suggestion